releases/handbook.md

```diff
@@ -1,193 +1,71 @@
+* Promote clarity on dates and phases so distributions, working groups, and the whole community can plan accordingly.
+* Improve release cycle by moving technical decisions affecting the release to the Release Management team including Kubeflow platform
+
+### Release Meetings and Communication
+* Create a new Release Management team on Slack for asynchronous communication within the team
+* Bi-weekly meetings during Software Development (Week 1-Week 8). Weekly meetings between week 9-12 as needed.
+* Meetings will focus on roadmap status, including blockers, feature discussions, the overall status of the release, and the help needed.
+
+### Release Documentation
+
+* Build a team of technical writers to be embedded in each working group for the release time frame. 
+* Provide roles within the community to boost participation and bring new members into this group.
+* Technical Documentation Lead will lead this team; technical documentation can include a blog and slides to announce the Release.
+
+### Timeline
+* Week 0 -  (Release and Roadmap discussions) WG Leads and Release Manager meet to discuss the roadmap planned for the Release
+* Week 2 - (Software development Phase) WG Leads/Liaisons meet to discuss any release challenges, release changes, and help needed from the community (to communicate on Kubeflow Community Meeting)
+* Week 4 - (Software development Phase) WG Leads/Liaisons meet to discuss any release challenges, release changes, and help needed from the community (to communicate on Kubeflow Community Meeting)
+* Week 6 - (Software development Phase) WG Leads/Liaisons meet to discuss any release challenges, release changes, and help needed from the community (to communicate on Kubeflow Community Meeting)
+* Week 8 - (Last week of Software development Phase) WG Leads/Liaisons meet to discuss any release challenges, release changes, and help needed from the community (to communicate on Kubeflow Community Meeting)
+* Week 9 - (Feature Freeze and Prep for Release) WG Leads/Liaisons meet to discuss any release challenges, release changes, and help needed from the community (to communicate on Kubeflow Community Meeting). Discuss items required to prepare for Release (including documentation).
+* Week 10 - (Community and Distribution testing starts) Users and distributions run their automated test suites and try to stay away from manual labor-intensive tests.
+* Week 11 - (Bug Fixing) - Discuss any potential blockers for the Release.
+* Week 12 - (Release) - Items to discuss: bug fixes required, release cut, documentation needed.
+- Manifests WG leads synchronizes and cut the release on Kubeflow Platform/Manifests.
+- The Release Manager is responsible for approving/reviewing the blog and slides announcing the release.
 
 ## People and Roles
 
-### Release Manager
-The Release Manager will be responsible for coordinating the release and taking ultimate accountability for all release tasks to be completed on time.
```

Comment:
why we don't have more requirements for the release manager.how this process will work and who'll be deciding/electing the new RM?

Reply:
Please check the current version and whether something is missing instead of what duplication might have been removed.